src/ng/compile.js

```diff
@@ -3638,7 +3638,7 @@ SimpleChange.prototype.isFirstChange = function() { return this.previousValue ==
 
 
 var PREFIX_REGEXP = /^((?:x|data)[:\-_])/i;
-var SPECIAL_CHARS_REGEXP = /[:\-_]+(.)/g;
+var SPECIAL_CHARS_REGEXP = /([:\-_]+(.))/g;
```

Comment:
Why the extra parenthesis? They are not needed afaict (and slow the RegExp down - theoretically :grinning:)